src/main/java/org/apache/maven/shared/filtering/DefaultMavenResourcesFiltering.java

```diff
@@ -321,19 +347,20 @@ public void filterResources(MavenResourcesExecution mavenResourcesExecution) thr
     }
 
     /**
-     * Get the encoding to use when filtering the specified file. Properties files can be configured to use a different
-     * encoding than regular files.
+     * Get the encoding to use when filtering the specified file. Properties files
+     * can be configured to use a different encoding than regular files.
      *
-     * @param file The file to check
-     * @param encoding The encoding to use for regular files
+     * @param file               The file to check
```

Comment:
Probably better not to try to line these up. It's too hard to maintain.
But we should make the initial letter lower case in keeping with Oracle guideline

Reply:
I use `spoltless:apply` to format the file according to project rules, so not fully sure how to proceed, do you want me to revert the format changes from spotles?

Reply:
Did spotless do that? That's new, i think, and a strike against spotless if so.

Reply:
I now reverted all unrelated javadoc changes